fixtures/transformers/transformer.js

```diff
@@ -0,0 +1,20 @@
+exports.transformers = [
+  {
+    name: "Test transformations with transfromer test id",
+    transform: (body, req, res) => {
+      console.log("Resource Path", req.method, req.url);
+      console.log(req.url);
```

Comment:
Does this line still need to be here?

Reply:
No, as one log is enough for both req method and url. Removing it